actionpack/test/controller/http_token_authentication_test.rb

```diff
@@ -79,12 +79,12 @@ def authenticate_long_credentials
     end
   end
 
-  test "authentication request with badly formatted header" do
-    @request.env['HTTP_AUTHORIZATION'] = "Token foobar"
+  test "authentication request with text" do
```

Comment:
Right, is there any reason that you didnt create a new test, and instead it is changing an existent one?